packages/@apphosting/adapter-nextjs/src/interfaces.ts

```diff
@@ -74,9 +74,29 @@ export interface RoutesManifest {
 
 // The output bundle options are specified here
 export interface OutputBundleOptions {
+  /**
+   * Path to the generated bundle.yaml file
+   */
   bundleYamlPath: string;
-  outputDirectory: string;
+  /**
+   *  Path to where the base output directory (.apphosting) is generated
```

Comment:
Is this output always called .apphosting?

Reply:
yeah its always .apphosting